src/utils/user_utils.js

```diff
@@ -22,9 +22,10 @@ export function displayUsername(
     user: UserProfile,
     teammateNameDisplay: string
 ): string {
-    let name = localizeMessage('channel_loader.someone', 'Someone');
+    let name = '';
 
     if (user) {
+        name = localizeMessage('channel_loader.someone', 'Someone');
```

Comment:
How does this work? It looks like 'Someone' will always get overridden here by the else of the if statement below